### PR TITLE
GUID + FMT + Version number

### PR DESCRIPTION
**Pull request by schnipschnap**

### CODE REVIEW

File: core/mondoo-windows-security.mql.yaml [hunk 1]
```diff
@@ -1032,8 +968,8 @@ queries:
             default: "Failure";
           }
     mql: |
-      auditpol.where(subcategory == props.auditpolFileShare).list != []
-      auditpol.where(subcategory == props.auditpolFileShare).all(inclusionsetting == props.auditpolSuccessFailure)
+      auditpol.where(subcategoryguid == "0CCE9244-69AE-11D9-BED3-505054503030").list != []
+      auditpol.where(subcategoryguid == "0CCE9244-69AE-11D9-BED3-505054503030").all(inclusionsetting == props.auditpolSuccessFailure)
```
Comment (atomic111) [inline core/mondoo-windows-security.mql.yaml:972]: the guid is wrong. it should be 0CCE922F-69AE-11D9-BED3-505054503030

File: core/mondoo-windows-security.mql.yaml [hunk 2]
```diff
@@ -646,8 +628,8 @@ queries:
             default: "Failure";
           }
     mql: |
-      auditpol.where(subcategory == props.auditpolAuditPolicyChange).list != []
-      auditpol.where(subcategory == props.auditpolAuditPolicyChange).all(inclusionsetting == props.auditpolSuccess || inclusionsetting == props.auditpolSuccessFailure)
+      auditpol.where(subcategoryguid == "6997984D-797A-11D9-BED3-505054503030").list != []
+      auditpol.where(subcategoryguid == "6997984D-797A-11D9-BED3-505054503030").all(inclusionsetting == props.auditpolSuccess || inclusionsetting == props.auditpolSuccessFailure)
```
Comment (atomic111) [inline core/mondoo-windows-security.mql.yaml:632]: i think the guid value is wrong. it should be 0CCE922F-69AE-11D9-BED3-505054503030

Comment (atomic111) [inline core/mondoo-windows-security.mql.yaml:632]: please adjust the guid to 0CCE922F-69AE-11D9-BED3-505054503030

File: core/mondoo-windows-security.mql.yaml (hunk 1 above)
Comment (schnipschnap) [reply core/mondoo-windows-security.mql.yaml:972]: Fixed in latest commit

Comment (schnipschnap) [reply core/mondoo-windows-security.mql.yaml:972]: Fixed in latest commit

**Review by atomic111 — APPROVED** [review]
@schnipschnap thank you for the improvement

**Pull request merged**